packages/ui/src/components/Input/index.module.scss

```diff
@@ -1,23 +1,33 @@
 @use '@/scss/underscore' as _;
 
+.wrapper {
+  position: relative;
+}
+
 .input {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-  padding: _.unit(3) _.unit(5);
+  width: 100%;
+  padding: _.unit(3) _.unit(10) _.unit(3) _.unit(5);
   border-radius: _.unit(2);
+  border: _.border();
   background: var(--color-control-background);
-  color: var(--color-heading);
+  color: var(--color-font-primary);
+  caret-color: var(--color-primary);
   font: var(--font-control);
   transition: var(--transition-default-control);
 
   &::placeholder {
-    color: var(--color-placeholder);
+    color: var(--color-font-tertiary-3);
   }
 
-  &:disabled {
-    background: var(--color-control-background-disabled);
-    color: var(--color-secondary);
+  &:focus {
+    border: _.border(var(--color-control-focus));
   }
 }
+
+.clearBtn {
+  position: absolute;
```

Comment:
do we need to use absolute positioning? how about flexbox?

Reply:
have to keep the fullwidth styling of the input field. As browsers autocomplete apply styling change on input field only. 

i.e 
<img width="1043" alt="image" src="https://user-images.githubusercontent.com/36393111/157250027-8f50394d-f862-4c5c-9432-195b50f749b7.png">

Reply:
need to double check, it has 1~2px height difference when focus / blur

Reply:
thx to @demonzoo  point out the css bug